ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/access-lists.j2

```diff
@@ -6,6 +6,9 @@
 
 {%     for access_list in access_lists | arista.avd.natural_sort %}
 #### {{ access_list }}
+{%       if access_lists[access_list].enable_counters is arista.avd.defined(true) %}
```

Comment:
New blank line required between if condition and the text. When ever the if condition is true then a new-line and the text are written to the doc file. So this adds a space between acess_list heading and the next command/text. 

Required to avoid md022 error.